docs/networking/ippool.md

```diff
@@ -34,6 +34,18 @@ To create a new IP pool:
 1. Go to the **Selector** tab to specify the **Scope** and **Priority** of the IP pool.
    ![](/img/v1.2/networking/ippool-scope.png)
 
+When you operate from Harvester UI directly, the `Scope` only includes `Namespace`, click `Add Scope` to add new items.
+
+### Create IP Pool from Rancher Manager UI
+
+When the Harvester cluster is imported to `Rancher Manager`, and you operate from `Rancher Manager UI > Virtualization Management` and locate to this page, it shows differently.
```

Comment:
this page, it shows differently. -> Network tab in the IP Pools section the page looks differently.